docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,240 @@
+.. _start_simple:
+
+Simple Example
+==============
+
+This is perhaps the most minimal Envoy configuration which can be used to validate basic plain HTTP
+proxying. It is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Quick Start
+-----------
+
+This Quick Start gets you started using only files in the Envoy repo. The sections below explain
+this configuration file and execution steps in detail.
+
+Copy both :repo:`configs/Dockerfile` and
```

Comment:
Can a clearer distinction be made between getting the docker container up and running, what the configuration means and playing around with envoy(sample curls, logs to look at, stats to look at)?